src/containers/paint-editor.jsx

```diff
@@ -54,21 +54,25 @@ class PaintEditor extends React.Component {
     render () {
         return (
             <PaintEditorComponent
+                name={this.props.name}
                 rotationCenterX={this.props.rotationCenterX}
                 rotationCenterY={this.props.rotationCenterY}
                 svg={this.props.svg}
```

Comment:
Whenever the paint editor gets new props, if it gets an SVG it would reload and clear the undo state as if you switched to a different costume.  I think we have to design a way to avoid this if you have to pass in name.

Reply:
Fixed in https://github.com/LLK/scratch-paint/pull/78/commits/138718008c7e78c3da3e15919f98e45825de544d